scripts/centos-req.sh

```diff
@@ -18,6 +18,7 @@ sudo yum install -y centos-release-scl
 sudo yum install -y devtoolset-8-make
 # install DTC
 sudo yum install -y dtc
+sudo yum install -y python
```

Comment:
A few lines above there is a python36, python36-pip and python36-devel installation. Is this python installation for an earlier or later version?

Reply:
It would ideally be for a later version. I added this because there was a bug I ran into with ubuntu-req.sh where one of the tests was looking for python2 instead of python3. I don't know a way to test on centos, so I'm not entirely sure if the bug would still occur there, so I added it just in case.

Reply:
which test was looking for python2?

Reply:
It was both of the tracegen tests.